client/src/components/SearchResult.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react'
```

Comment:
I suppose we can wait to rename all the `Search*` files until there's other types of search results, but this is another file whose name doesn't seem specific enough. Could be `SerachResourcesResult.js`.

Reply:
Yeah thats a fair point, the grouping is because this is for the `/search` or global search - it probably should be `ResourceSearchResult.js` so it is consistent with `ResourceDetails` etc